src/main/java/net/minestom/server/entity/Metadata.java

```diff
@@ -246,6 +246,36 @@ public void write(@NotNull BinaryWriter writer) {
             //noinspection unchecked
             return (Value<T>) value.withValue(reader);
         }
+
+        @NotNull
+        public Value<T> getMetaValue() {
```

Comment:
Is it used anywhere? Or a rebase artifact?

Reply:
Artifact, will fix in next commit